src/app/cli/src/init/transaction_snark_profiler.ml

```diff
@@ -32,7 +32,9 @@ let create_ledger_and_transactions num_transactions :
       let public_key = Public_key.compress k.public_key in
       let account_id = Account_id.create public_key Token_id.default in
       Mina_ledger.Ledger.create_new_account_exn ledger account_id
-        (Account.create account_id (Currency.Balance.of_int 10_000)) ) ;
+        (Account.create account_id
+           (Currency.Balance.of_uint64
+              (Unsigned.UInt64.of_int64 Int64.max_value) ) ) ) ;
```

Comment:
1/2 of max value maybe? would cause an overflow when an account only receives/ receives in the first transaction.

Reply:
`Currency.Balance.t` is an unsigned 64-bit value, so `Int64.max_value` is only half of that.

Reply:
oops, didn't notice that.